local-cli/server/middleware/getDevToolsMiddleware.js

```diff
@@ -14,8 +14,8 @@ const launchChrome = require('../util/launchChrome');
 
 const {exec} = require('child_process');
 
-function launchChromeDevTools(port, args = '') {
-  var debuggerURL = 'http://localhost:' + port + '/debugger-ui' + args;
+function launchChromeDevTools(host, args = '') {
+  var debuggerURL = 'http://' + host + '/debugger-ui' + args;
```

Comment:
Why eslint does not throw error when using "var" ?

Reply:
There's still a lot of files that are not converted to es6 so the rule is not enabled.